concourse/pipelines/sap-workload-test.yaml

```
@@ -0,0 +1,78 @@
+jobs:
+- name: sap-workload-test
+  plan:
+  - get: sap-hana-post-script-gcs
+    trigger: false
+  - task: generate-post-script
+    config:
+      platform: linux
+      type: docker-image
+      source:
+        repository: google/cloud-sdk
+        tag: latest
+      run:
+        path: sh
+        args:
+        - -exc
+        - |
+          # TODO - versioning/id. Want to map these logs for the check-post-script-results task
+          cat "gsutil cp /var/log/messages gs://test-bucket-for-terraform/log_output" | sap_post_script.sh
+          gsutil cp sap_post_script.sh gs://test-bucket-for-terraform/sap_post_script.sh
+
+  - task: run-sap-tf-environment
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: hashicorp/terraform
+          tag: latest
+      run:
+        path: sh
+        args:
+        - -exc
+        - |
+          # TODO - will need a unique instance_name for parallel runs
+        - cat <<EOT >> sap_hana.tf
+          module "sap_hana" {
+          source = "https://storage.googleapis.com/cloudsapdeploy/terraform/latest/terraform/sap_hana/sap_hana_module.zip"
+          project_id = "pneil-sandbox"
+          zone = "us-east1-b"
+          machine_type = "n1-highmem-32"
+          subnetwork = "default"
+          linux_image = "rhel-8-4-sap-ha"
+          linux_image_project = "rhel-sap-coud"
+          instance_name = "hana-instance"
+          post_deployment_script = "gs://test-bucket-for-terraform/sap_post_script.sh"
+          }
+          EOT
+        - terraform apply -auto-approve;
+          # TODO - handle crash here. What should we do if the plan is applied but never destroyed?
+          # Maybe just always destroy before apply?
+        - terraform destroy -auto-approve;
+
+  - task: check-post-script-results
+    config:
+      platform: linux
+      type: docker-image
+      source:
+        repository: google/cloud-sdk
+        tag: latest
+      run:
+        path: sh
+        args:
+        - -exc
+        - |
+          if gsutil cat gs://test-bucket-for-terraform/log_output | grep "ERROR - Deployment Exited" | grep -v grep; then
+            return 1
+          else
+            return 0
+          fi
```

still think this should be output from the job, but a few fixes.
1. `grep -q` instead of `| grep -v grep`
2. `return` is only valid in a function context
3. we set exit on errors with the `-exc` args; so we only need to run a command that fails
4. swap the logic - if we don't see success, it's a failure

```suggestion
          gsutil cat gs://test-bucket-for-terraform/log_output | grep -q "SUCCESS"
```